main.go

```diff
@@ -38,10 +40,20 @@ func main() {
 		log.Fatalf("Error converting nostr privkey to pubkey: %v", err)
 	}
 
-	db, err := gorm.Open(postgres.Open(cfg.DatabaseUri), &gorm.Config{})
-	if err != nil {
-		log.Fatalf("Failed to open DB %v", err)
+	var db *gorm.DB
+	if strings.HasPrefix(cfg.DatabaseUri, "postgres://") || strings.HasPrefix(cfg.DatabaseUri, "postgresql://") || strings.HasPrefix(cfg.DatabaseUri, "unix://") {
```

Comment:
I've never used `unix://` for a PG connection, does it work with GORM? But ok for me.

Reply:
https://github.com/getAlby/lndhub.go/blob/main/db/db.go#L21